include/odp/api/spec/crypto.h

```diff
@@ -252,6 +274,27 @@ typedef enum {
 	 */
 	ODP_AUTH_ALG_ZUC_EIA3,
 
+	/** AES_CMAC in EIA2 mode */
+	ODP_AUTH_ALG_AES_CMAC_EIA2,
```

Comment:
Similar comment:
1) Move before ODP_AUTH_ALG_ZUC_EIA3
2) CMAC not needed in enum name: ODP_AUTH_ALG_AES_EIA2 is enough
3) Mention 128-EIA2 in comment (similarly to 128-EIA3 reference of ZUC)

There's bug in current documentation of ODP_AUTH_ALG_SNOW3G_UIA2: 128-EEA1 in heading vs. 128-EIA1 in body text.

Reply:
Ok